FrontEnd/src/main/java/resourceDisplay/AssetsRepository.java

```diff
@@ -0,0 +1,313 @@
+package resourceDisplay;
+
+import com.google.cloud.spanner.*;
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * This Class does all DB access
+ */
+public class AssetsRepository {
+    /**
+     * This is just for testing the function that returns assets by type
```

Comment:
I don't understand this comment

Reply:
I was using it for the temporary filter by kind, now that we have the new one I deleted this function